rskj-core/src/test/java/co/rsk/peg/bitcoin/BitcoinTestUtils.java

```diff
@@ -37,4 +38,14 @@ public static Address createP2SHMultisigAddress(NetworkParameters networkParamet
 
         return Address.fromP2SHScript(networkParameters, outputScript);
     }
+
+    public static Sha256Hash createHash(int nHash) {
```

Comment:
For what exactly is this method used?

Reply:
That change is not part of my changes, sorry, the PR needed to be rebased.